threedigrid_builder/interface/db.py

```diff
@@ -497,6 +502,10 @@ def get_cross_section_locations(self) -> CrossSectionLocations:
                     models.CrossSectionLocation.bank_level,
                     models.CrossSectionLocation.friction_type,
                     models.CrossSectionLocation.friction_value,
+                    models.CrossSectionLocation.vegetation_stem_density,
```

Comment:
It is singular here, but plural in `db.py`. Please pick one.

Reply:
I'm not sure if I get it correctly.
Plurals are written in 'Definition' and Singulars in 'Location'.